scripts/issue/497/config_gas_station.py

```diff
@@ -0,0 +1,50 @@
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main():
+    safe = GreatApeSafe(registry.eth.badger_wallets.techops_multisig)
+    station = interface.IGasStationExact(
+        registry.eth.badger_wallets.gas_station, owner=safe.account
+    )
+
+    labels, addrs, min_bals, min_top_ups = [], [], [], []
+    for label, addr in registry.eth.badger_wallets.items():
+        min_bal = 2e18
+        min_top_up = .1e18
+        if not label.startswith('ops_') or 'multisig' in label:
+            continue
+        if 'executor' in label:
+            min_bal = 1e18
+        if label == 'ops_botsquad':
+            min_bal = 5e18
+        if label == 'ops_botsquad_cycle0':
```

Comment:
do you think we should store the custom configs in a dict somewhere?

Reply:
yea we had something like that in the original refill script, but being able to already cut out so many of the old addresses, it looks like the exceptions are getting less and less. also there is dealing with both `min_bal` and `min_top_up`, basically making it more work to intro a dict then to write these three ifs.

tbh im hoping we can move towards deprecating the botsquad accounts as well and replace them by chainlink, making this gas station only for non bot eoas (deployers and executors).